config_system/config_system/general.py

```diff
@@ -25,7 +25,7 @@
 
 def can_enable(depends):
     value = expr.condexpr_value(depends)
-    if value == 'y':
+    if value is True:
```

Comment:
Missed an equivalent change in the is_visible() function just below.

Looking at these functions though, it's a bit odd that we take the condition as an argument instead of the config

Reply:
Maybe we could simplify these two functions to just return `value` directly? (and maybe error if the type is wrong)

I don't mind if you do that here or in a follow up though

Reply:
Yeah. I've a feeling that we rely on the explicit check. For example where condexpr_value returns the value of a string identifier. So this is definitely a follow-up candidate.